FpsCheck.py

```diff
@@ -243,7 +243,12 @@ def is_eTLD_Plus1(self, site):
         Returns:
             boolean with truth value dependent on value of get_public_suffix
         """
-        return self.etlds.get_public_suffix(site, strict=True) is not None
+        assert site is not None
+        site = site.lstrip("https://")
```

Comment:
```suggestion
        site = site.removeprefix("https://")
```
[`lstrip`](https://docs.python.org/3/library/stdtypes.html#str.lstrip) isn't what you want here, since it removes all the leading characters from the string if they are in the argument passed in. So, `"https://shhhhh.com".lstrip("https://")` gives `".com"`, rather than the intended `"ptths.com"`.

Reply:
Changed it back to `replace`

Reply:
Can we use `removeprefix` instead of `replace`? It's unlikely to be an issue here, but `replace` would remove substrings in the middle of the string, which is never something we want this to do. So `removeprefix` conveys our intent more clearly.